core-plugins/src/main/java/co/cask/hydrator/plugin/common/KinesisUtil.java

```diff
@@ -24,22 +24,17 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.concurrent.TimeUnit;
+
 /**
  * Utility methods to help create and manage Kinesis Streams
  */
 public final class KinesisUtil {
 
   private static final Logger LOG = LoggerFactory.getLogger(KinesisUtil.class);
-  private static final long DELETE_TIMEOUT = 1000 * 120;
-  private static final long CREATE_TIMEOUT = 3 * 60 * 1000;
+  private static final long TIMEOUT = 3 * 60 * 1000;
```

Comment:
Making it configurable might be an overkill since it is needed for creation of stream and has nothing to do with data writing operations. If you suggest we can make it configurable